packages/flutter/lib/src/painting/box_painter.dart

```diff
@@ -395,9 +395,11 @@ class BoxShadow {
 
   final double spreadRadius;
 
-  // See SkBlurMask::ConvertRadiusToSigma().
-  // https://github.com/google/skia/blob/bb5b77db51d2e149ee66db284903572a5aac09be/src/effects/SkBlurMask.cpp#L23
-  double get _blurSigma => blurRadius * 0.57735 + 0.5;
+  /// The [blurRadius] in sigmas instead of logical pixels.
+  ///
+  /// See SkBlurMask::ConvertRadiusToSigma().
+  /// <https://github.com/google/skia/blob/bb5b77db51d2e149ee66db284903572a5aac09be/src/effects/SkBlurMask.cpp#L23>
+  double get blurSigma => blurRadius * 0.57735 + 0.5;
```

Comment:
this should probably point to MaskFilter.blur.

I think I'd move the link to skia's docs into a regular comment and only have the dartdoc point to MaskFilter.